src/components/_expansion-card.scss

```
@@ -52,6 +78,16 @@ category: Components
   box-sizing: border-box;
   color: var(--color-label);
   width: 100%;
+  &.-three-line {
+    .ncgr-expansion-card__trailing {
+      margin: 0 0 auto 1rem;
+    }
+  }
+  &.-dense {
+    .ncgr-expansion-card__header {
+      padding: 0.5rem 1rem;
+    }
+  }
 }
 
 .ncgr-expansion-card__item {
```

ここも名前をlist-viewの方に揃えてもいいかも

```suggestion
.ncgr-expansion-card__list-tile {
```